companies/1st-online-solutions-limited.json

```diff
@@ -18,5 +18,5 @@
         "https://github.com/datenanfragen/data/pull/1189"
     ],
     "suggested-transport-medium": "email",
-    "quality": "verified"
+    "quality": "tested"
```

Comment:
🚫 **[companylint]** <[tested-requirements](https://github.com/datenanfragen/data/issues/811)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record has `quality` of `tested` but doesn't specify `required-elements`.

Preferably set `required-elements`, otherwise demote to `verified`.
```suggestion
    "quality": "verified"
```